src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -163,13 +168,17 @@ describe('<PillButton> component', () => {
             expect(getButtonByName('This pill is in use up arrow')).toHaveClass(
                 'PillButton__pill PillButton__pill--isOpen PillButton__pill--isActive'
             );
+            expect(screen.queryByText(upArrowLabel)).not.toBeInTheDocument();
+            expect(screen.queryByText(clearLabel)).not.toBeInTheDocument();
         });
 
         it('should trigger toggle state once when button is clicked', async () => {
             const user = userEvent.setup();
             await user.click(getButtonByName(upArrowLabel));
 
             expect(toggleDropdownMock).toHaveBeenCalledTimes(1);
+            expect(screen.queryByText(upArrowLabel)).not.toBeInTheDocument();
+            expect(screen.queryByText(clearLabel)).not.toBeInTheDocument();
```

Comment:
I don't think we need to repeat these tests here, if we had them in the previous `it`

Reply:
:+1: